### DISCUSSION TITLE
High cpu load and logs not working.

### DISCUSSION THREAD

**Discussion by rajnishnpst** [fluent/fluent-bit#5717] [Q&A]
We are facing high cpu load 15-50% on our production server on td-agent-bit and when i try to enable log service not starts.
looked into /var/log/messages but not found anything related.

**Comment by agup006** [comment]
Try running fluent bit manually, if using td-agent-bit use the following: /opt/td-agent-bit/bin/td-agent-bit -c <PATH TO YOUR CONFIG>

Also a note that td-agent-bit has been deprecated as of 1.9, and now only fluent-bit packages are maintained
